Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM composer:2.0 as composer
+
+WORKDIR /usr/local/src/
+
+COPY composer.lock /usr/local/src/
+COPY composer.json /usr/local/src/
+
+RUN composer install --ignore-platform-reqs --optimize-autoloader \
+    --no-plugins --no-scripts --prefer-dist
+    
+FROM phpswoole/swoole:php8.0-alpine
```

Comment:
Let's use our own base alpine image where we have better control on what's going on 🙏